library/src/scripts/embeddedContent/StoryExampleDropDownMessages.tsx

```diff
@@ -0,0 +1,366 @@
+/*
+ * @author Stéphane LaFlèche <stephane.l@vanillaforums.com>
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import React from "react";
+import { titleBarClasses } from "@library/headers/titleBarStyles";
+import { t } from "@library/utility/appUtils";
+import MessagesCount from "@library/headers/mebox/pieces/MessagesCount";
+import MessagesContents from "@library/headers/mebox/pieces/MessagesContents";
+import { uniqueIDFromPrefix } from "@library/utility/idUtils";
+import DropDown, { FlyoutType } from "@library/flyouts/DropDown";
+import classNames from "classnames";
+import Frame from "@library/layout/frame/Frame";
+import FrameHeaderWithAction from "@library/layout/frame/FrameHeaderWithAction";
+import LinkAsButton from "@library/routing/LinkAsButton";
+import { ButtonTypes, buttonUtilityClasses } from "@library/forms/buttonStyles";
+import { ComposeIcon } from "@library/icons/titleBar";
+import FrameBody from "@library/layout/frame/FrameBody";
+import FrameFooter from "@library/layout/frame/FrameFooter";
+import MeBoxDropDownItemList from "@library/headers/mebox/pieces/MeBoxDropDownItemList";
+import { IMeBoxMessageItem, MeBoxItemType } from "@library/headers/mebox/pieces/MeBoxDropDownItem";
+
+interface IProps {
+    buttonClassName?: string;
+    className?: string;
+    contentsClassName?: string;
+    toggleContentClassName?: string;
+    countClass?: string;
+}
+
+interface IState {
+    open: boolean;
+}
+
+const messagesData = [
+    {
+        authors: [
+            {
+                userID: 25,
+                name: "Joe With Spaces",
+                photoUrl: "https://dev.vanilla.localhost/applications/dashboard/design/images/defaulticon.png",
```

Comment:
Is there any way to make these URLs relative to Storybook and not reliant on Vanilla also running alongside it?

Reply:
Most other images in Storybook seem to be linking to an external CDN, which is probably better than making them relative. My primary concern here is the additional requirement of a web server running on the same host as Storybook to serve up these requests to `dev.vanilla.localhost`. If we need to use relative URLs as an interim solution or something, I'm cool with that.